presentation/src/main/java/com/anytypeio/anytype/presentation/search/GlobalSearchViewModel.kt

```diff
@@ -343,6 +477,13 @@ class GlobalSearchViewModel(
             override val isLoading: Boolean
         ): ViewState()
 
+        data class RelatedInit (
```

Comment:
let's add a todo for converting current `Related` class to sealed class, and put `RelatedInit` inside this class and rename `RelatedInit` to `Init` ?

Reply:
done